05/index.php

```diff
@@ -3,5 +3,16 @@
 $menu = [
     ['label' => 'home', 'url' => '/'],
     ['label' => 'about me', 'url' => '/about'],
-    ['label' => 'contact', 'url' => 'contact'],
-];
\ No newline at end of file
+    ['label' => 'contact', 'url' => '/contact'],
+];
+
+$items = $menu;
+
+function generateMenu($items) {
+    foreach ($items as $item) {
+        echo $item['label']. " " . $item['url'] . "<br>";
```

Comment:
Menu to przynajmniej elementy `a` - a najlepiej struktura `ul > li > a`.
Tutaj możesz doczytać: https://devmentor.pl/b/jak-stworzyc-trzypoziomowe-drop-down-menu-w-html-i-css